public/index.html

```diff
@@ -5733,7 +5733,7 @@ <h2 class="nm-48 white">
             document.getElementById("address-form"),
             document.getElementById("hero-address-entry"),
           ],
-          googlePublicApiKey: "AIzaSyB0o_nPI-xjHYKg7KB0bl87Yhnf2ng9Nsg",
+          googlePublicApiKey: "AIzaSyAmQVT2CAy_KGQ1uutPJ2DPf93TcaxboS4",
```

Comment:
You can run `git checkout main -- public/index.html && git commit --all -m "Removes api key change" && git push` to get rid of this change.